scipy/linalg/flapack_gen.pyf.src

```diff
@@ -237,6 +237,24 @@ subroutine <prefix>getc2(n,a,ipiv,jpiv,info)
 
 end subroutine <prefix>getc2
 
+subroutine <prefix>gesc2(n,lu,rhs,ipiv,jpiv,scale)
+    ! x,scale = gesc2(lu,rhs,ipiv,jpiv,overwrite_b=0)
+    ! Solve  A * X = scale * RHS
+    ! A = P * L * U * Q
+    threadsafe
+    callstatement {int i;for(i=0;i\<n;++ipiv[i],++jpiv[i++]);(*f2py_func)(&n,lu,&n,rhs,ipiv,jpiv,&scale);for(i=0;i\<n;--ipiv[i],--jpiv[i++]);}
+    callprotoargument int*,<ctype>*,int*,<ctype>*,int*,int*,<ctypereal>*
```

Comment:
I tried `<ctype2>` per your notes @ilayn but I got an error. Is `<ctype2>` a valid shorthand?

Reply:
I changed it to `ctypereal` because I got an error with `ctype2`. I thought it said  `ctype2` was undefined, but maybe not.